modepy/tools.py

```diff
@@ -474,23 +474,43 @@ def estimate_lebesgue_constant(n, nodes, visualize=False):
     lebesgue_worst = np.sum(np.abs(eq_to_out), axis=1)
     lebesgue_constant = np.max(lebesgue_worst)
 
-    if visualize:
+    if visualize and dims == 2:
```

Comment:
Should this raise if visualize is `True` for other dimensions?

Reply:
Yeah, probably.